### DISCUSSION TITLE
My status page get 404

### DISCUSSION THREAD

**Discussion by odera89** [upptime/upptime#481] [Q&A]
Hi, i dont know why, but my status page has 404. Just 404 words. can somebody help me why?

https://upptime.truthweb.com/

My .upptimerc.yml

`owner: TruthWebDesign # Your GitHub organization or username, where this repository lives
repo: upptime # The name of this repository

sites:
  - name: Google
    url: https://google.com
    
status-website:
  cname: upptime.truthweb.com
  logoUrl: https://raw.githubusercontent.com/upptime/upptime.js.org/master/static/img/icon.svg
  name: Upptime
  introTitle: "**Upptime** is the open-source uptime monitor and status page, powered entirely by GitHub."
  introMessage: This is a sample status page which uses **real-time** data from our [GitHub repository](https://github.com/upptime/upptime). No server required — just GitHub Actions, Issues, and Pages. [**Get your own for free**](https://github.com/upptime/upptime)
  navbar:
    - title: Status
      href: /
    - title: GitHub
      href: https://github.com/$OWNER/$REPO`

**Comment by RavelloH** [comment]
Try [this](https://upptime.js.org/docs/faq/#im-getting-a-404-error-in-setup-ci).
